src/pages/productCategories/ProductCategories.less

```diff
@@ -0,0 +1,42 @@
+#productCategory {
+    padding: 20px;
+
+    .ant-card-body {
+        padding: 0;
+    }
+
+    #bg {
+        background-color: black;
+    }
+
+    #card {
+        width: 218px;
+        margin: 20px 20px 20px 0;
+        border-radius: 20px;
+    }
+
+    img {
+        border-radius: 20px;
+    }
+
+    .mask {
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        position: absolute;
+        background-color: rgba(0, 0, 0, 0.267);
+        border-radius: 20px;
+    }
```

Comment:
you can try add 
`
.mask {
display: flex;
justify-content: center;
align-item: center;
}
`
so it make title center x and y
if not working try change title from `<p id="title">{category.name}</p>` to `<div class="title">{category.name}</div>`

Reply:
tidak berfungsi, malah menjadi ancur

Reply:
hover dan font oke